agent/recorder-agent.c

```diff
@@ -511,11 +527,15 @@ gboolean
   return TRUE;
 }
 
+/* *INDENT-OFF* */
 gboolean
-hwangsae_recorder_agent_recorder_interface_handle_url
+    hwangsae_recorder_agent_recorder_interface_handle_url
```

Comment:
I guess you've accidentally added the spaces you want to avoid here.

Reply:
Yes you are totally right. Thanks for checking.